atest/testdata/standard_libraries/builtin/verify.robot

```diff
@@ -74,6 +74,10 @@ Should Not Be Equal with bytes containing non-ascii characters
     Should Not Be Equal    ${BYTES WITH NON ASCII}    unicode
     Should Not Be Equal    ${BYTES WITH NON ASCII}    ${BYTES WITH NON ASCII}
 
+Should Be Equal With Case Insensitivity
+    Should Be Equal     "test value"     "TEST VALUE"     ignore_case=True
+    Should Be Equal As Strings  "test value"     "TEST VALUE"     ignore_case=True
```

Comment:
- Quotes in the verified text are not necessary and mainly confuse reader.
- I'd prefer having case-insensitive test after normal Should Be Equal test. 
- Should test also with non-ASCII characters. For example that `HYVÄÄ YÖTÄ` (good night in Finnish) equals `hyvää yötä` case-insensitively. Could consider using `[Template]` like many other tests here.
- Should Be Equal As Strings should get its own test after normal Should Be Equal As Strings test.